apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Getting started
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
```

Comment:
I'll give a slightly more specific example for here, as we mention using the Provider at application root.

```
import { HopperProvider } from "@hopper-ui/components";
import { createRoot } from 'react-dom/client';

const root = createRoot(document.getElementById('root'));
root.render(
  < HopperProvider >
    <App />
  </HopperProvider >
);
```